src/main.ts

```diff
@@ -1,10 +1,12 @@
-interface Person {
-    name: string;
-    age: number;
-}
+import { birthday } from './person';
```

Comment:
These changes are gibberish because you're copying files instead of merging, don't do this in the future please.